src/main/java/com/knighttodo/knighttodo/gateway/privatedb/repository/RoutineRepository.java

```diff
@@ -7,6 +7,6 @@
 
 public interface RoutineRepository extends JpaRepository<Routine, String> {
 
-    @Query("select r from Routine r where r.templateId = r.id")
+    @Query("from Routine r left join fetch r.todos where r.templateId = r.id")
```

Comment:
Do you really need left join fetch here? It is only to avoid adding @Transactional in some tests, isn't it?

Reply:
It's to be able to fetch todos from Routine template and then create copies of todos in the copied routine.

Reply:
Are you sure that it does not work without join fetch?

Reply:
removed it at all, can't even remember why I did it previously, but supposedly it could be changed to @transactional